sdk/template/tests.yml

```diff
@@ -5,12 +5,13 @@ stages:
     parameters:
       AllocateResourceGroup: false
       ServiceDirectory: template
-      TestSamples: true
+      MatrixReplace:
+        - TestSamples=.*/true
```

Comment:
For my own clarity, this is telling your script to find/replace in the value of TestSamples with `True`. This is due to the fact that it's set to `False` by default for the other invocations of `ci.tests.yml`. Correct?

Reply:
Correct.